util.js

```diff
@@ -60,6 +60,29 @@ exports.getTestModifier = (node, mod) => {
 	return testModifiers.find(property => property.name === mod);
 };
 
+/**
+ * Removes a given test-modifier from the source surrounding the given node
+ *
+ * @param {string} params.modifier - Name of the modifier
+ * @param {Node} params.node - ESTree-node as provided by Eslint
+ * @param {Context} params.context - ESLint-context as provided
+ *
+ * @return {Array} Compound parameters to be used as arguments for `fix.replaceTextRange()`
+ */
+exports.removeTestModifier = params => {
+	const modifier = params.modifier.trim();
+	const range = exports.getTestModifier(params.node, modifier).range.slice();
+	const replacementRegExp = new RegExp(`\\.|${modifier}`, 'g');
+	const source = params.context.getSourceCode().getText();
+	let dotPosition = range[0] - 1;
+	while (source.charAt(dotPosition) !== '.') {
+		dotPosition -= 1;
+	}
+	let snippet = source.substring(dotPosition, range[1]);
```

Comment:
Can you use `slice` instead of `substring`? I know it was like this already, but better for consistency. I always use `slice` over `substring`.

https://github.com/sindresorhus/eslint-plugin-unicorn/issues/65

Reply:
Of course - i am happily copying (most of) your habits.. 😸